src/composables/auth.ts

```diff
@@ -59,26 +60,50 @@ export const useAuth = createCustomScopedComposable(() => {
   const isAuthenticating = ref(false);
   const isMnemonicRestored = ref(false);
   const encryptionKey = ref<CryptoKey | null>();
+  const isMnemonicMigrationCheckDone = ref(false);
 
-  const [mnemonicEncrypted, mnemonic] = useSecureStorageRef<string>(
+  /**
+ * Deprecated storage key for the mnemonic.
+ * Used for migration from the old storage key to the new one.
+ */
+  const oldMnemonic = useStorageRef<string>(
```

Comment:
Such things should be managed by the migration.
I think this is really bad for us to have such deprecations in the code.